pipenv/core.py

```diff
@@ -2303,6 +2303,9 @@ def do_run(command, args, three=None, python=False, pypi_mirror=None):
     inline_activate_virtual_environment()
     try:
         script = project.build_script(command, args)
+        cmd_string = ' '.join([script.command] + script.args)
+        if environments.is_verbose():
+            click.echo(crayons.normal("Run: {0}".format(cmd_string)), err=True)
```

Comment:
I have one small request -- in order to be consistent across our output elsewhere, the ideal output format would be something to the effect of:
```suggestion
            click.echo(crayons.normal("$ {0}".format(cmd_string)), err=True)
```